<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinSetupWindow</class>
 <widget class="QDialog" name="JoinSetupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="panel" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="nickEdit">
        <property name="placeholderText">
         <string>Digite seu nome</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hostIpEdit">
        <property name="placeholderText">
         <string>Ex: 127.100.001</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="portSpin"/>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
